.github/workflows/test_qml.yaml

```diff
@@ -165,20 +167,19 @@ jobs:
       - name: Add msbuild to PATH
         uses: microsoft/setup-msbuild@v1.1
 
-      - name: Install glean depedencies
+      - name: Install depedencies
         shell: bash
         run: |
           pip3 install -r requirements.txt
-          python3 scripts/utils/generate_glean.py
 
-      - name: Importing translation files
+      - name: Building tests
         shell: bash
         run: |
           export PATH=/c/MozillaVPNBuild/bin:$PATH
-          python3 scripts/utils/import_languages.py -m
+          mkdir -p build
+          cmake -S . -B $(pwd)/build
+          cmake --build $(pwd)/build --config Release --target qml_tests
 
       - name: Running tests
         shell: bash
-        run: |
-          export PATH=/c/MozillaVPNBuild/bin:$PATH
-          ./scripts/tests/qml_tests.sh
```

Comment:
does it mean we can remove this script when macos is done?

Reply:
Yep!

Theoretically we could remove it now as the current state of MacOS+CMake is that it builds the executable just fine, but the app bundle is woefully incomplete. The unit tests likely work right now if built with CMake.